doc-content/enterprise-only/release-notes/rn-770-fixed-issues-ref.adoc

```diff
@@ -0,0 +1,72 @@
+[id='rn-770-fixed-issues-ref']
+= Fixed issues in {PRODUCT} 7.7.0
+
+{PRODUCT} 7.7.0 provides increased stability and fixed issues listed in this section.
+
+ifdef::PAM[]
+[NOTE]
+====
+{PRODUCT_DM} fixed issues apply to {PRODUCT_PAM} as well. For a list of {PRODUCT_DM} 7.7.0 fixed issues, see the https://access.redhat.com/documentation/en-us/red_hat_decision_manager/7.7/html-single/release_notes_for_red_hat_decision_manager_7.6/index[_Release Notes for Red Hat Decision Manager 7.7_]
+====
+
+== {CENTRAL}
+
+* When you add a deployment unit and manually enter GAV values, the runtime strategy is not set to the configured default value [https://issues.redhat.com/browse/RHPAM-2623[RHPAM-2623]]
+* When you save a filter and then activate it, an error message displays [https://issues.redhat.com/browse/RHPAM-2545[RHPAM-2545]]
+* In the guided rule editor, you cannot use the combination of complex values [https://issues.redhat.com/browse/RHPAM-2457[RHPAM-2457]]
+* The state of a {KIE_SERVER} is not updated in the server template after the server disconnects and reconnects to {CENTRAL} [https://issues.redhat.com/browse/RHPAM-2544[RHPAM-2544]]
+* Guided rule editor removes `matches` operator from rule file [https://issues.redhat.com/browse/RHPAM-2631[RHPAM-2631]]
+* The legacy test scenario fails if you try to validate a list of strings [https://issues.redhat.com/browse/RHPAM-2492[RHPAM-2492]]
+
+== Process Designer
+
+* If the process designer Properties panel is open and you click the *Maximize* button, it closes and restores the panel [https://issues.redhat.com/browse/RHPAM-2613[RHPAM-2613]]
+* Validation for a signal name fails if the name includes spaces, colons (:), or other special characters [https://issues.redhat.com/browse/RHPAM-2557[RHPAM-2557]]
+* When you create a gateway, no options are available from the *Default Route* drop-down list in the *Properties* panel [https://issues.redhat.com/browse/RHPAM-2536[RHPAM-2536]]
+
+== Process instance migration
+
+* The process instance migration service does not work with an Oracle database [https://issues.redhat.com/browse/RHPAM-2558[RHPAM-2558]]
+
+== Process engine
+
+* {CENTRAL} fails to display process details when you configure multiple {KIE_SERVERS} using Smart Router [https://issues.redhat.com/browse/RHPAM-2568[RHPAM-2568]]
+* When you try to reopen a closed case, it does not resumes from the point at which it was closed rather it creates a new process instance [https://issues.redhat.com/browse/RHPAM-2556[RHPAM-2556]]
```

Comment:
When you try to reopen a closed case, the case does not open and a new process instance appears

I might have changed the meaning. Check.